frontend/src/v5/ui/routes/viewer/drawings/drawingsList/drawingsList.styles.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ *  Copyright (C) 2024 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+import { CardList } from '@components/teamspace/teamspaceList/teamspaceList.styles';
+import { TableVirtuoso } from 'react-virtuoso';
+import styled from 'styled-components';
+
+export const VirtualisedList = styled(TableVirtuoso)`
```

Comment:
Do you see any problem with using a virtualised list for  cardlist? That way we could solve any potential performance issues in any other card that would use a a list

Reply:
Virtualising tickets list is a problem, at least given the current state of the code. In fact, we have to either include the filtering section in the list or not:
- if we do not include it, it will break our current design. In fact, instead of scrolling with the list, it stays at the top all the time, taking away room from the list
![image](https://github.com/3drepo/3drepo.io/assets/43547703/5fbec86a-4567-4ec8-96ec-d37b5b40c1e2)

- if we do include it, we have to pass it as a prop to the virtual list. Since it is passed as a prop to the virtual list, a change in the search will force the virtual list to re-render and the input will lose its focus


https://github.com/3drepo/3drepo.io/assets/43547703/8804f6d7-26d1-4713-9751-a7babba5757c